environment.yaml

```diff
@@ -6,7 +6,7 @@ dependencies:
   - python>3.6
   - astropy>=4.2.1
   - astropy-healpix
-  - astroquery
+  - astroquery>=0.4.4
```

Comment:
We should add this to the CI yamls as well. And while you're editing this file, can you also update the h5py requirement to `h5py>=3.0`? That change was made in PR #1097 but this file was missed in that PR.

Reply:
I've fixed the h5py verisons (in f52ad1a6 and cbc2e4cf). As of right now, there is only one astroquery dependency listed in the CI yamls, which is under `ci/pyuvdata_tests.yaml`. Comically enough, this is _already_ listed as `astroquery>=0.4.4`, so there was nothing to do there. I could add that to the other yamls as well, but none of the other CI builds seem to include the optional dependencies (which astroquery is listed as), so I'd just like to double-check with you to make sure that's what you really want.

Reply:
Ah, I didn't realize it was already set to 0.4.4 in our main CI yaml. I think it should also be added to the `pyuvdata_tests_windows` one but none of the others. The windows one should have all the optional dependencies that are supported on Windows. Thanks!